src/nets/e2e_asr_attctc_th.py

```diff
@@ -354,6 +355,33 @@ def recognize(self, x, recog_args, char_list, rnnlm=None):
             self.train()
         return y
 
+    def recognize_batch(self, xs, recog_args, char_list, rnnlm=None):
```

Comment:
Same comment.
You should replace the original `recognize` to this `recognize_batch`